jobs/build-cosa.Jenkinsfile

```diff
@@ -87,6 +87,13 @@ node {
     shortcommit = commit.substring(0,7)
 }
 
+// HACK: for a period of time we're going to be hacking on the RHCOS
+// pipeline older streams and we've created some branches in the COSA
+// git repo that we'd like to build and push only to the staging quay repo.
+if ((gitref.startsWith("rhcos-") && gitref.endsWith("-new")) {
+    params.CONTAINER_REGISTRY_REPO = params.CONTAINER_REGISTRY_STAGING_REPO
```

Comment:
Sadly (but likely for the best), Jenkins will not let you change the value of a parameter.

Reply:
hmm. could have sworn I'd done that before.. oh well - new upload incoming.

Reply:
Me too actually, but I actually tested it this morning for different reasons and hit that limitation.